swagger.yaml

```diff
@@ -172,6 +174,20 @@ definitions:
           - full
           - delta
 
+  storage-configuration:
```

Comment:
Please nest s3-specific configuration to allow for easy extension with other storage types.